includes/class-eddcoursegiftingcronmod.php

```diff
@@ -56,14 +56,14 @@ public static function get_instance() {
 	public function __construct() {
 		add_filter( 'cron_schedules', array( $this, 'wdm_cron_cchedules' ) );
 		add_action( 'emails_gift_reminder', array( $this, 'email_gift_reminder' ) );
-		add_action( 'gift_emails_handler', array( $this, 'gift_emails_handler' ) );
+		add_action( 'gift_emails_handler', array( $this, 'gift_email_handler' ) );
 	}
 
 	public function wdm_cron_cchedules( $schedules ) {
-		if ( ! isset( $schedules['wdm_daily_emails_gift_reminder'] ) ) {
-			$schedules['wdm_daily_emails_gift_reminder'] = array(
-				'interval' => 86400,
-				'display' => __( 'Once a day', 'learndash-gift-edd' ),
+		if ( ! isset( $schedules['wdm_emails_gift_reminder'] ) ) {
+			$schedules['wdm_emails_gift_reminder'] = array(
+				'interval' => 1800,
+				'display' => __( 'After 30 min', 'learndash-gift-edd' ),
```

Comment:
I recommend adding a name that represents the interval and not the function. So, maybe just remove "After" so it is "30 minutes".

Reply:
Changed